### PR TITLE
[Player 4652] [Android][VRSampleApp] Fullscreen doesn't work

### PR DESCRIPTION
**Pull request by SergeyBicarte**
This fix have two parts. First one is for android , it's introduce back saving/restoring original (started) size of layout. Second one is for  React Native, it's seems style "flex": 1 is not working correctly in some conditions for main react native view (as result, some times, that view have wrong size), i propose setting sizes in style with "width" and "height" attribs.

### CODE REVIEW

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java [hunk 1]
```diff
@@ -6,28 +6,30 @@
...
   private FrameChangeCallback frameChangeCallback;
 
   private WindowManager windowManager;
   private boolean fullscreen = false;
 
+  private int sourceWidth, sourceHeight;
+
   public interface FrameChangeCallback {
-      void onFrameChangeCallback(int width, int height,int prevWidth,int prevHeight);
+    void onFrameChangeCallback(int width, int height,int prevWidth,int prevHeight);
```
Comment (isheftel-ooyala) [inline sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java]: code style here, spaces between arguments

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java [hunk 2]
```diff
@@ -78,6 +78,7 @@ public void createSubViews() {
 
     final View decorView = ((Activity)getContext()).getWindow().getDecorView();
     decorView.setOnSystemUiVisibilityChangeListener(this);
+
```
Comment (isheftel-ooyala) [inline sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java]: redundant new  line

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java [hunk 3]
```diff
@@ -92,15 +93,22 @@ public FrameLayout getAdView() {
     return playerFrame;
   }
 
+  @Override
+  protected void onAttachedToWindow () {
```
Comment (isheftel-ooyala) [inline sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java]: code style

**Review by isheftel-ooyala — CHANGES_REQUESTED** [review]

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java (hunk 1 above)
Comment (SergeyBicarte) [reply sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java]: Done

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java (hunk 2 above)
Comment (SergeyBicarte) [reply sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java]: Done

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java (hunk 3 above)
Comment (SergeyBicarte) [reply sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java]: Done

**Review by isheftel-ooyala — APPROVED** [review]
LGTM

**Review by bychkovdmitry — APPROVED** [review]
LGTM

**Pull request merged**